### PR TITLE
clean up error handling and include count of live_nodes when they change

### PR DESCRIPTION
**Pull request by ianrose14**

### CODE REVIEW

File: solrman/main/solrman/solrman.go [hunk 1]
```diff
@@ -101,7 +102,7 @@ func run(logger *log.Logger) error {
 
 	mutex, err := acquireAndMonitorRedisMutex(smLogger, redisPool)
 	if err != nil {
-		errors.New(fmt.Sprintf("Failed to acquire Solrman redis mutex: %s", err))
+		return fmt.Errorf("Failed to acquire Solrman redis mutex: %s", err)
```
Comment (ianrose14) [inline solrman/main/solrman/solrman.go]: I'm assuming lack of return here is an oversight...

Comment (dragonsinth) [inline solrman/main/solrman/solrman.go]: Wow yep

**Comment by dragonsinth** [comment]
@PratikDeoghare don't forget you have a lot of changes you need to port to this repo.  @ianrose14 will need to update our `go get` version of this and he's going to run into local mods.

**Comment by ianrose14** [comment]
@PratikDeoghare how should I proceed?  I'm happy to just sit on this for a bit so you have time to port your changes back to the main repo, and then we can `go get` both of our changes into `mn` repo

**Comment by PratikDeoghare** [comment]
I understand the porting part. What is `go get`?

**Comment by dragonsinth** [comment]
It's how we pull third-party repo changes into our own codebase

**Pull request merged**